internal/gitlab/gitlab.go

```diff
@@ -1900,3 +1906,18 @@ func CreateMergeRequestCommitDiscussion(project string, mrID int, sha string, ne
 	note := discussion.Notes[0]
 	return fmt.Sprintf("%s/merge_requests/%d#note_%d", p.WebURL, note.NoteableIID, note.ID), nil
 }
+
+// hasNextPage is a pagination helper to handle the case where the API
+// response comes without the HTTP X-Total(-Page) header. Reference:
+// https://docs.gitlab.com/ee/user/gitlab_com/index.html#pagination-response-headers
+func hasNextPage(resp *gitlab.Response) (int, bool) {
+	if resp.TotalPages != 0 {
+		if resp.CurrentPage >= resp.TotalPages {
+			return 0, false
+		}
+	} else if resp.CurrentPage >= resp.NextPage {
```

Comment:
So suppose there are 100 pages ... what will the 99th resp.NextPage return?  0?

Reply:
I see @zampierilucas asked the same question.  I'll resolve this and you can answer it once.

Reply:
`X-Page` (`CurrentPage`) starts at 1, so when `CurrentPage == 100` the `X-Next-Page` will be empty in the HTTP header and the go-gitlab lib will make it `0`.